scripts/004-gcc-stage2.sh

```diff
@@ -33,25 +33,29 @@ rm -rf build-$TARGET-stage2 && mkdir build-$TARGET-stage2 && cd build-$TARGET-st
   --quiet \
   --prefix="$PS2DEV/$TARGET_ALIAS" \
   --target="$TARGET" \
+  --with-headers="$PS2DEV/$TARGET_ALIAS/$TARGET/include" \
   --enable-languages="c,c++" \
   --with-float=hard \
+  --with-headers=no \
```

Comment:
--with-headers is defined twice.

The last one should be removed becouse in the second stage build we use the headers from newlib.

Reply:
Fixed